### DISCUSSION TITLE
Free Tier Face Matching For POC

### DISCUSSION THREAD

**Discussion by mlourencoeb** [ballerine-io/ballerine#1953] [Ideas]
Hey everyone,

I am co-founder of [Youverse](https://www.youverse.id/). We build face authentication services and we would love to have our products integrated in ballerine.

I am writing here specifically to ask it would make sense to offer free trials for developer that want to build POCs with face matching during the KYC process.

Would that something valuable for this community?

Best,
/M

**Comment by SpekGit** [comment]
I think it is good to have face recognition based on ID during KYC. It needs to be verified with founder of Ballerine
